### ISSUE TITLE
can't connect to fortmatic

### ISSUE THREAD

**Issue by dkent600** [daostack/alchemy#2003]
Only tried on rinkeby.  Trying to get the networkId in `util.getNetworkId(web3Provider)`:

`web3Provider.send("net_version", (error, response)...`

on the provider as returned by Web3Modal.  But I'm getting the following error, and, being unhandled, the promise is not invoking the reject callback.

```
Uncaught DOMException: Failed to execute 'postMessage' on 'Window': (error, response) => {
                        if (error) {
                            reject(error);
     ...<omitted>... } could not be cloned.
```

Result is that no error is reported by Alchemy, and can't connect to fortmatic.

**Comment by dkent600** [comment]
Hmm, same with Portmatic.  May be our own error.

**Comment by dkent600** [comment]
OK, Portmatic, if it works at all only works on mainnet where 2.0 currently has no presence.

I have tested with Metamask, Formatic, Torus and WalletConnect (Gnosis Safe)

**Comment by dkent600** [comment]
Fixed in 2.0
